CHANGELOG.md

```diff
@@ -47,6 +47,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 ### Infrastructure
 * Add CHANGELOG ([#342](https://github.com/opensearch-project/dashboards-maps/pull/342))
+* Add maps metrics api from saved object ([#362](https://github.com/opensearch-project/dashboards-maps/pull/362))
```

Comment:
we should better frame this feature.

Reply:
Some of options, WDYT?
* Add stats API for maps and layers in maps plugin
* Add endpoint to retrieve maps and layers statistics in maps plugin
* Get maps and layers stats in maps plugin

Also feel free to give other suggestions, thanks!

Reply:
> * Add stats API for maps and layers in maps plugin

This should work.